### DISCUSSION TITLE
Kernel panic on Tanix w2

### DISCUSSION THREAD

**Discussion by wzooroo** [devmfc/debian-on-amlogic#103] [General]
What could be the problem Kernel panic on Tanix w2? In boot.config I selected box=tanixw2 and others with w2 CPU. Nothing helps(

![IMG20241020143914](https://github.com/user-attachments/assets/01d8d1a9-b222-4c75-aaeb-63640266d9f3)

**Comment by favianrizqulloh** [comment]
Same issue here. But after commenting again the line `box=tanixw2`, the box can boot.

uncommented:
![WIN_20250527_09_34_49_Pro](https://github.com/user-attachments/assets/33cb1b64-cfc6-42ef-9c82-24c62ae16827)

commented:
<video src="https://github.com/user-attachments/assets/83ab0deb-184c-4cda-b56a-c7a1e439cc8a" width="640" height="360" controls></video>

Any updates?
